### DISCUSSION TITLE
Merge Redis support PR

### DISCUSSION THREAD

**Discussion by n5nk** [JanusGraph/janusgraph#4124] [General]
Hi,

Can the maintainer review Redis PR for Janusgraph https://github.com/JanusGraph/janusgraph/issues/3018 this would be helpful for everyone running Atlas in distributed mode.

**Comment by li-boxuan** [comment]
I left a review last year but haven't heard back from the authors since then. If you are desperately in need of it, try forking their branch and build a custom version of JanusGraph.

**Reply by n5nk** [reply]
Thanks @li-boxuan will do

**Reply by n5nk** [reply]
@li-boxuan , Need small help to understand the PR. What is the purpose of [redisIndexKeys](https://github.com/JanusGraph/janusgraph/pull/3100/files#diff-38e3c3344a732e0178ef639cda424154cadc782a6eb321969d57087219d68dfcR90) here in the PR

**Reply by li-boxuan** [reply]
@n5nk I myself haven't got a chance to review that PR thoroughly. I suggest you ping the original authors.
